src/github.org

```diff
@@ -0,0 +1,104 @@
+#+TITLE: GitHub Training 2017
+#+AUTHOR: Sahit Chintalapudi and Josh Ting 
+#+EMAIL: schintalapudi@gatech.edu
+#+REVEAL_THEME: black
+#+REVEAL_TRANS: linear
+#+REVEAL_SPEED: fast
+#+REVEAL_PLUGINS: (notes pdf)
+#+REVEAL_HLEVEL: 1
+#+OPTIONS: toc:nil timestamp:nil reveal_control:t num:nil reveal_history:t tags:nil author:nil
+
+* What is Git?
+- Git is a "Distributed Version Control System" (DCVS)
+- It is the tool RoboJackets uses to collaborate on code
+- Git tracks changes made to files in your project, and handles merging
+  these changes
+- Working on different parts of code and then combining them creates unique challenges that
+  cannot be handled by file sharing systems like DropBox and Google Drive
+
+* What is GitHub?
+- GitHub is a publically hosted git server
+- It hosts our projects as "repositories" on the internet
```

Comment:
There's a lot of terminology that newcomers won't understand when teaching git (like repository), maybe you could consolodate it all into [a table](https://github.com/RoboJackets/robocup-training/blob/master/src/2016/git.org#git-vocabulary)?